runtime/src/lib.rs

```diff
@@ -424,6 +425,20 @@ impl_runtime_apis! {
 			<Runtime as pallet_evm::Trait>::FeeCalculator::min_gas_price()
 		}
 
+		fn account_code_at(address: H160) -> Vec<u8> {
```

Comment:
Those existed before right?
They have been removed when we moved away from the runtime API is that right ?

Reply:
Yes that's exactly right. We're adding them back now because:
* They are harmless even if we aren't using them
* It keeps our code closer to stock frontier
* It prepares us for when https://github.com/paritytech/frontier/pull/199 lands upstream